docs/index.yml

```diff
@@ -109,7 +109,7 @@ conceptualContent:
         - url: https://dotnet.microsoft.com/download
```

Comment:
```suggestion
        - url: https://dotnet.microsoft.com/download/dotnet-core/3.1
```

Reply:
The problem with the suggestion is that it will need to be updated for each release. Since "Download .NET" is the first giant box on the page, how about we change this link back to https://docs.microsoft.com/en-us/dotnet/core/sdk? (But as `core/sdk.md`.)

Reply:
![image](https://user-images.githubusercontent.com/24882762/80628901-ec78d480-8a06-11ea-9da4-ceb50518a897.png)

Reply:
Good idea 👍